### DISCUSSION TITLE
i can't see the network wifi available,im on linux on laptop in the pre installation on virtual box

### DISCUSSION THREAD

**Discussion by jrm567** [GameServerManagers/LinuxGSM#4230] [General]
im not able to see the network available i tried a lot of things,still not working can someone know the solution ? when i click "connect wifi card" on the vm on the adatapter one(bridging i selected my wifi card,someone know how to fix it ? thanks !  version ok kali linux(debian 64 bit kali-linux-2023.2-virtualbox-amd64 pre installation of virtual box) im not able to see any network i can just see the ethernet connections (lo ,eth0) 
![Capture d'écran 2023-06-09 224337](https://github.com/GameServerManagers/LinuxGSM/assets/135761446/0f5932bc-a8b4-4ad6-bd53-1fcc10292692)

**Comment by dgibbs64** [comment]
This forum is for LinuxGSM and has nothing to do with Kali Linux
